components/raftstore/src/store/peer.rs

```diff
@@ -3541,43 +3538,12 @@ where
         Ok(Either::Left(propose_index))
     }
 
-    fn execute_transfer_leader<T>(
+    pub fn execute_transfer_leader<T>(
         &mut self,
         ctx: &mut PollContext<EK, ER, T>,
         msg: &eraftpb::Message,
         peer_disk_usage: DiskUsage,
     ) {
-        // log_term is set by original leader, represents the term last log is written
-        // in, which should be equal to the original leader's term.
-        if msg.get_log_term() != self.term() {
```

Comment:
This is still necessary.

Reply:
This check is moved to `on_transfer_leader_msg`.

Reply:
I mean it's also necessary for follower.

Reply:
Yes. Follower should also call `on_transfer_leader_msg` when receiving `TransferLeader` msg.